tests/e2e/login_test.go

```diff
@@ -17,7 +19,6 @@ var _ = Describe("odoLoginE2e", func() {
 	// Comand related constants
 	const baseOdoLoginCommand = "odo login"
 	const baseOdoProjectCreate = "odo project create"
-	const baseOdoProjectDelete = "odo project delete -f "
 	const ocWhoamiCommand = "oc whoami"
```

Comment:
I feel the tests would read better without these constants and variables (i.e. if the commands were inlined in the code directly).

Reply:
I can remove them no problem.

I did that because the same command was being used in multiple places and it would make it easier to update the command,  should anything change.

Reply:
Yep, I understand but I think it'd be better to be more explicit in this case… :)

Reply:
Updated locally, pushing shortly